keyboards/ubest/vn/info.json

```
@@ -0,0 +1,174 @@
+{
+    "keyboard_name": "vn",
+    "manufacturer": "ubest",
+    "url": "",
+    "maintainer": "mfkiiyd",
+    "usb": {
+        "vid": "0x6D66",
+        "pid": "0x0868",
+        "device_version": "0.0.1"
+    },
+    "layout_aliases": {
+        "LAYOUT": "LAYOUT_all"
+    },
+    "layouts": {
+        "LAYOUT_all": {
```

`LAYOUT_all` is generally used with keyboards support multiple layouts.
```suggestion
    "layouts": {
        "LAYOUT": {
```